v2/index.html

```diff
@@ -2122,10 +2122,47 @@ <h5>Repeated measures</h5>
             </aside>
         </section>
         <section>
-            <h5>Rhythmic sequence</h5>
+            <h5>Duration Group</h5>
             <p>
-                When the same rhythmic sequence is repeated, the sequence of rhythmic values can be stated once
-                before the note names.
+                Multiple duration values MAY be declared together. These values form a Duration Group, and MAY be
+                used to indicate repeating rhythmic patterns for the notes that follow.
+            </p>
+            <p>
+                The number and order of duration values MUST apply to an equivalent number of notes that follow.
+                If the number of notes that follow exceeds the number of duration values given in the Duration
+                Group, the duration values MUST repeat from the first.
```

Comment:
I don't understand this sentence.

Reply:
It just means that the duration cycle... `484ABCDEFGAB` would result in the three repetitions of the `484` figure.

Reply:
I am confused with the use of `MUST` here. My understanding is that it is about how things must be coded, not how what is written must match the code, so the `MUST` is confusing in this paragraph.

Reply:
It is like if you say "a note following a `4` MUST be a quarter note"

Reply:
It could a) stop (i.e., not repeat the durations) or b) go backwards (weird, I know, but it *could*) or c) be interruptible by adding a specific duration (which is why the requirement to cancel it is necessary) or d) apply to only otherwise grouped notes (e.g., only beams would take the duration group). 

It's better to be pedantic than to be vague, IMO. There's no room for interpretation when is says it MUST repeat from the first.

BTW, we do say that a note following a 4 MUST be a quarter note... https://plaine-and-easie.info/v2/index.html#durations

Reply:
> BTW, we do say that a note following a 4 MUST be a quarter note...

Can you point me to the line? Sorry, I cannot find it.